csrankings-x.csv

```diff
@@ -245,6 +248,7 @@ Xinchao Wang,Stevens Institute of Technology,https://sites.google.com/site/sitex
 Xindong Wu 0001,University of Louisiana - Lafayette,http://www.ucs.louisiana.edu/~xxw8007,X8sHmqIAAAAJ
 Xinfeng Ye,University of Auckland,https://unidirectory.auckland.ac.nz/profile/x-ye,NOSCHOLARPAGE
 Xing Gao 0001,University of Delaware,http://xgao-work.github.io,Wkw_2-wAAAAJ
+Xing Wei,Xi'an Jiaotong University,http://gr.xjtu.edu.cn/web/weixing,KNyC5EUAAAAJ

```

Comment:
Add suffix to disambiguate (see https://dblp.org/pid/14/4301.html).

Reply:
Need to update, remove for now.